extensions/ui/org.eclipse.smarthome.ui.basic/snippets/switch.html

```diff
@@ -2,13 +2,17 @@
 	<span class="mdl-form__icon">
 		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%" />
 	</span>
-	<div class="mdl-form__label">
+	<span %labelstyle% class="mdl-form__label">
 		%label%
-	</div>
+	</span>
+	<span %valuestyle% class="mdl-form__value mdl-form__value--group">
```

Comment:
All `.mdl-form__value` elements have class `font-weight: 700;` style. For me it looks just like any other value.